app/models/campaign.js

```diff
@@ -10,6 +10,13 @@ var CampaignSchema = new Schema({
   learn_more: {type: String, trim: true},
   publish: {type: Boolean, trim: true},
   influencer: {type: String, trim: false}
+}, {
+  timestamps: {
+    createdAt: 'created_at',
+    updatedAt: 'updated_at'
+  }
 });
 
-mongoose.model('Campaign', CampaignSchema);
+var Campaign = mongoose.model('Campaign', CampaignSchema);
```

Comment:
There's not really a difference at this point (in this PR). `module.exports` is just a way to expose the "API" of the module to other modules. It was mostly in preparation for a future PR. I can pull it out of this one if you prefer and add it to the next one.